src/sentry/static/sentry/app/views/organizationActivity/activityFeed.jsx

```diff
@@ -92,27 +94,18 @@ class ActivityFeed extends React.Component {
       body = <LoadingError onRetry={this.fetchData} />;
     } else if (this.state.itemList.length > 0) {
       body = (
-        <div className="activity-container">
-          <ul className="activity">
-            {this.state.itemList.map(item => {
-              try {
```

Comment:
Not sure why this try/catch is needed since `ErrorBoundary` will report to Sentry.